sync/sync_head.go

```diff
@@ -160,7 +160,7 @@ func (s *Syncer[H]) incomingNetworkHead(ctx context.Context, head H) error {
 }
 
 // verify verifies given network head candidate.
-func (s *Syncer[H]) verify(ctx context.Context, newHead H) (bool, error) {
+func (s *Syncer[H]) verify(ctx context.Context, newHead H) (isSoft bool, _ error) {
```

Comment:
what's the point of naming the return vals here?

Reply:
To say exactly what this `bool` means. It's not clear (at all) from doc-comment. I'm fine to add a line to doc-comment and leave result param unnamed.

Reply:
Done